File: remote/src/main/java/com/kelsos/mbrc/domain/Artist.java
package com.kelsos.mbrc.domain;

public class Artist {
  private long id;
  private String name;

  public Artist(long id, String name) {
    this.id = id;
    this.name = name;
  }

  public long getId() {
    return id;
  }

  public String getName() {
    return name;
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/domain/Artist.kt
package com.kelsos.mbrc.domain

import com.kelsos.mbrc.data.db.RemoteDatabase
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.QueryModel
import com.raizlabs.android.dbflow.structure.BaseQueryModel

@QueryModel(database = RemoteDatabase::class)
class Artist() : BaseQueryModel() {
  @Column var id: Long = 0
  @Column var name: String = ""
}
